recipes-web/iot-rest-api-server/iot-rest-api-server.bb

```diff
@@ -27,8 +27,7 @@ SYSTEMD_AUTO_ENABLE_${PN} = "disable"
 USERADD_PACKAGES = "${PN}"
 GROUPADD_PARAM_${PN} = "-r restful"
 USERADD_PARAM_${PN} = "\
---system --home ${localstatedir}/lib/empty \
---no-create-home --shell /bin/false \
+--system --create-home --shell /bin/false \
```

Comment:
the commit message says iotivity-node installs to home but did you mean to say iot-rest-api-server?

with useradd-staticids.bblass used in Ostro, the home dir is picked from ostro-passwd and is defined as ```/var/lib/empty```. Was the problem that iot-rest-api-server was not able to create that dir but now that it gets created here it's OK? Perhaps explain the problem in the commit message.